dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -1413,7 +1413,6 @@ class DevToolsMemoryTest {
       'global',
       'activate',
       'devtools',
-      '0.2.5',
```

Comment:
Should we leave it off, or re-pin to 2.0.0 and kick the can down the road another year?

Reply:
We need to pin. To keep it up to date, we would need to autoroll.

Reply:
Done.

Reply:
FYI devtools 2.0.0 has not been migrated to null safety